tracks/serializers.py

```diff
@@ -11,17 +11,24 @@ class Meta:
         fields = ["name", "url"]
 
 
-class TrackSerializer(serializers.ModelSerializer):
+class ClientTrackSerializer(serializers.ModelSerializer):
     sources = SourceSerializer(many=True, read_only=True)
 
     class Meta:
         model = Track
-        fields = ["label", "track_id", "colour", "trigger", "display_order", "on_by_default", "additional_info", "description", "sources"]
+        fields = ["label", "track_id", "colour", "trigger", "type",
+            "display_order", "on_by_default", "additional_info", "description", "sources"]
+
+class BackendTrackSerializer(serializers.ModelSerializer):
```

Comment:
We are already in Backend. If this very specific to GenomeBrowser then GenomeBrowserTrackSerialiser or something would be more appropriate.